pyfakefs/patched_packages.py

```diff
@@ -94,7 +95,12 @@ def __getattr__(self, name):
             return getattr(self._xlrd_module, name)
 
 
-if parsers is not None:
+# From pandas v 1.2 onwards the python fs functions are used even when the engine selected is "c".
+# This means that we don't explicitly have to change the engine.
+patch_pandas = (
```

Comment:
There are two other places above with use `if parsers is not None` - these also have to be adapted to use `patch_pandas` instead.

Reply:
Got it, let me update.

Reply:
Updated.